pkg/eval/sys/sys.go

```diff
@@ -0,0 +1,25 @@
+// Package sys exposes variables and functions that deal with the specific
+// system being run on. Such as the OS name and CPU architecture.
+package sys
+
+import (
+	"runtime"
+
+	"github.com/elves/elvish/pkg/eval"
+	"github.com/elves/elvish/pkg/eval/vars"
+)
+
+//elvdoc:var arch
```

Comment:
It's worth documenting which constants in the Go standard library these correspond to by linking to https://pkg.go.dev/runtime?tab=doc#pkg-constants for both constants.